pkg/pool-cow/contracts/CowRouter.sol

```diff
@@ -0,0 +1,349 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+import { ICowRouter } from "@balancer-labs/v3-interfaces/contracts/pool-cow/ICowRouter.sol";
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+import {
+    AddLiquidityKind,
+    AddLiquidityParams,
+    SwapKind,
+    VaultSwapParams
+} from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+
+import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/FixedPoint.sol";
+
```

Comment:
```suggestion
```
Also nice to sort each section by decreasing length, for readability.

Reply:
I skipped lines for different packages, which helps me see if the library comes from the vault, solidity-utils, etc. Should I group different packages in the same block of imports?